recipes/pySW4/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pySW4" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "b14bb605da99ffb557fe1742914207beb876602bb69576b95fc1176304b6c58a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/shaharkadmiel/pySW4/archive/v0.3.0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - obspy 1.*
+    - numpy
+    - scipy
+    - matplotlib
+    - gdal 2.*
+
+test:
+  imports:
```

Comment:
You should actually import something here. :)  At a minimum check that `pySW4` imports correctly.

Since there are some command-line scripts, you could add a `commands` section too, with entries like `pySW4-plot-image --help`, `pySW4-create-plots --help`, `png2mp4 --help` just to test that the scripts got set up properly.